app/paths.js

```diff
@@ -156,6 +157,9 @@ module.exports = {
   },
   generateRoute: require(path.join(__dirname, '/utils/generate_route.js')),
   requestToGoLive: {
-    index: '/service/:externalServiceId/request-to-go-live'
+    index: '/service/:externalServiceId/request-to-go-live',
```

Comment:
Suggestion above about making these return a function which takes the `externalId` which returns the correct String, rather than having to use `replace` later on.

Reply:
as discussed we are not changing this, please see the comment above, thanks